rts/Sim/Units/UnitTypes/Builder.cpp

```diff
@@ -553,7 +549,7 @@ void CBuilder::Update()
 void CBuilder::SlowUpdate()
 {
 	if (terraforming)
-		mapDamage->RecalcArea(tx1, tx2, tz1, tz2);
+		mapDamage->RecalcArea(tx1 - 3, tx2 + 3, tz1 - 3, tz2 + 3);
```

Comment:
consider a named constant instead of the magic 3. for example `CBuilder::TERRA_SMOOTHING_RADIUS` if I understand the meaning of the value right

Reply:
Good catch. Fixed.